rosetta-source/src/main/rosetta/product-qualification-func.rosetta

```diff
@@ -720,7 +721,10 @@ func Qualify_EquityOption_PriceReturnBasicPerformance_SingleName: <"Qualifies a
             and optionUnderlier -> Observable -> Asset -> Instrument -> Security only exists 
                     // qualifies that no feature other than averaging exists
             and (economicTerms -> payout -> optionPayout -> feature is absent
-                or economicTerms -> payout -> optionPayout -> feature -> averagingFeature only exists)
+                or economicTerms -> payout -> optionPayout -> feature -> averagingFeature only exists
+                or economicTerms -> payout -> optionPayout -> feature -> fxFeature only-element -> quanto  exists
+                or economicTerms -> payout -> optionPayout -> feature -> fxFeature only-element -> composite  exists)
```

Comment:
Why are these 2 additional assertions (for quanto and composite) already there in CDM 5, but not in CDM 6? Are we sure we also need to add them in CDM 6?